src/rpc/blockchain.cpp

```diff
@@ -716,9 +717,13 @@ UniValue getblock(const UniValue& params, bool fHelp)
             "  \"version\" : n,         (numeric) The block version\n"
             "  \"merkleroot\" : \"xxxx\", (string) The merkle root\n"
             "  \"finalsaplingroot\" : \"xxxx\", (string) The root of the Sapling commitment tree after applying this block\n"
-            "  \"finalorchardroot\" : \"xxxx\", (string) The root of the Orchard commitment tree after applying this block.\n"
-            "                                        Omitted for blocks prior to NU5 activation. This will be the null\n"
-            "                                        hash if this block has never been connected to a main chain.\n"
+            "  \"finalorchardroot\" : \"xxxx\", (string, optional) The root of the Orchard commitment tree after\n"
+            "                               applying this block. Omitted for blocks prior to NU5 activation. This\n"
+            "                               will be the null hash if this block has never been connected to a\n"
+            "                               main chain.\n"
+            "                               NB: The serialized representation of this field returned by this method\n"
+            "                                   was byte-flipped relative to its representation in the `getrawtransaction`\n"
+            "                                   output in prior releases up to v5.2.0. This has now been rectified.\n"
```

Comment:
```suggestion
            "                                   output in prior releases up to and including v5.2.0. This has now been\n"
            "                                   rectified.\n"
```
("up to" is ambiguous.)